app/js/components/drugOrderEntry/styles.scss

```diff
@@ -0,0 +1,73 @@
+.drug-order-entry {  
```

Comment:
I think you should add the license at the top of all the files

Reply:
Done! @jomadoye